lib/Client.js

```diff
@@ -1919,13 +1919,14 @@ class Client extends EventEmitter {
     /**
     * Get a guild's data via the REST API. REST mode is required to use this endpoint.
     * @arg {String} guildID The ID of the guild
+    * @arg {Boolean} [withCount=false] Whether the guild object will have approximateMemberCount and approximatePresenceCount
     * @returns {Promise<Guild>}
     */
-    getRESTGuild(guildID) {
+    getRESTGuild(guildID, withCount = false) {
         if(!this.options.restMode) {
             return Promise.reject(new Error("Eris REST mode is not enabled"));
         }
-        return this.requestHandler.request("GET", Endpoints.GUILD(guildID), true).then((guild) => new Guild(guild, this));
+        return this.requestHandler.request("GET", Endpoints.GUILD(guildID), true, withCount).then((guild) => new Guild(guild, this));
```

Comment:
I'm not seeing how passing a boolean as the request body passed any of your local tests. Should be passing `with_counts` as the parameter.